src/page-view.ts

```diff
@@ -1,6 +1,7 @@
 import * as utils from './utils'
```

Comment:
Small thing from the last PR - do we need to import * ? Why not scope the vars (helps in the future to understand unused parts)

Reply:
Hmm it might be needed based on how I set up the mocking in the tests, I'll play around with it some more

Reply:
Fixed